debian/control

```diff
@@ -3,7 +3,7 @@ Section: net
 Priority: extra
 Maintainer: George Kargiotakis <kargig@noc.grnet.gr>
 Build-Depends: debhelper (>= 7.0.50~), iproute2, iptables, arptables, ebtables,
-               vzctl, dnsutils, bridge-utils, vlan
+               vzctl, dnsutils, bridge-utils, vlan, iputils-arping
```

Comment:
`iputils-arping` should be added as a dependency. Not just as build dependency. I'll fix that with a new commit.